src/app/features/data-import/data-import.service.ts

```diff
@@ -55,17 +57,45 @@ export class DataImportService {
     });
   }
 
-  async importCsvContentToDB(csv: string): Promise<void> {
+  async importCsvContentToDB(csv: string, importMeta: ImportMetaData): Promise<void> {
```

Comment:
Is there a reason why you dont just work with 3 function parameters?

Reply:
I did, but I expected further information which are passed later. E.g. the parsed file, line numbers, if the file as the _id property etc. which then is encapsulated in the ImportMetaData - Felt like the correct way to pass those information